phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/utils/LibraryDetailsPanels.java

```diff
@@ -0,0 +1,5 @@
+package org.phoenicis.javafx.components.library.utils;
+
+public enum LibraryDetailsPanels {
```

Comment:
Maybe `LibraryDetailsPanelType` would be clearer.

Reply:
Done